src/app/[locale]/(pages)/accedi/errore/page.tsx

```diff
@@ -103,10 +103,7 @@ const LoginErrorPage = () => {
           </Grid>
           {typeof errorCode === 'string' && errorCode === '1001' ? (
             <Grid item xs={12} textAlign={'center'}>
-              <Button
-                onClick={() => pushWithLocale(process.env.NEXT_PUBLIC_URL_IO || '')}
-                variant="contained"
-              >
+              <Button onClick={() => pushWithLocale(ROUTES.LOGIN)} variant="contained">
```

Comment:
This is not explained in the PR description.

Reply:
After the change, I had to adjust the pages that used the absolute path by using the 'pushwithlocale' function and the relative path